traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -175,7 +176,7 @@ func SnapshotHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	api.CreateChangeLogRawTx(api.ApiChange, "Snapshot of CRConfig and Monitor performed for "+cdn, inf.User, inf.Tx.Tx)
+	api.CreateChangeLogRawTx(api.ApiChange, fmt.Sprintf("CDN: %v, ID: %v, ACTION: Snapshot of CRConfig and Monitor", cdn, inf.IntParams["id"]), inf.User, inf.Tx.Tx)
```

Comment:
probably want to avoid using fmt in this situation and especially `%v`.  It has to use reflection to fill in the string and and Sprintf doesn't add that much clarity...